python/src/context.rs

```diff
@@ -95,6 +95,29 @@ impl ExecutionContext {
         ))
     }
 
+    fn register_memtable(
```

Comment:
considering it's not taking an actual memtable as input, but record batches, i think `register_record_batches` would have been a better name.

Reply:
Renamed to register_record_batches